webserver/views/api.py

```diff
@@ -37,7 +38,13 @@ def get_low_level(mbid):
     try:
         return Response(load_low_level(mbid, offset), content_type='application/json')
     except NoDataFoundException:
-        raise NotFound
+        raise webserver.exceptions.InvalidAPIUsage("Not found", status_code=NotFound.code)
+
+@api_bp.route("/<string:notanmbid>/low-level", methods=["GET"])
+@crossdomain()
+def get_low_level_404(notanmbid):
+    """Specific low-level view for non-mbid url parts to return a json error message"""
+    raise webserver.exceptions.InvalidAPIUsage("Not found", status_code=NotFound.code)
```

Comment:
Do we need these views to catch a case when non-UUID is being submitted to the API? I guess `<uuid:mbid>` check just raises regular exception.

That's kind of ugly. :confused: What if we remove all validation on the path level and do it manually?